pkg/internal/checkup/trex/client_test.go

```diff
@@ -40,6 +40,38 @@ const (
 	portIdx = trex.SourcePort
 )
 
+func TestClearStatsSuccess(t *testing.T) {
+	expecter := expecterStub{expectTrexConsoleFailure: false}
+	c := trex.NewClient(expecter, trafficGeneratorPacketsPerSecond, testDuration, verbosePrintsEnabled)
+
+	_, err := c.ClearStats()
+	assert.NoError(t, err, "ClearStats returned an error")
+}
+
+func TestClearStatsFailure(t *testing.T) {
+	expecter := expecterStub{expectTrexConsoleFailure: true}
+	c := trex.NewClient(expecter, trafficGeneratorPacketsPerSecond, testDuration, verbosePrintsEnabled)
+
+	_, err := c.ClearStats()
+	assert.ErrorContains(t, err, "trex command \"clear\" failed. check logs for more information")
```

Comment:
Why is the expected error here shorter than the one in `TestStartTrafficFailure`?

Reply:
The reason is that we are mentioning the command that failed. The "clear" command is shorter than the "run" commmand, so it produces a shorter error message.